app/src/ui/branches/branches-container.tsx

```diff
@@ -261,4 +267,46 @@ export class BranchesContainer extends React.Component<
   ) => {
     this.setState({ selectedPullRequest })
   }
+
+  private onRenameBranch = (branchName: string) => {
+    const branch: Branch | undefined = this.props.allBranches.find(
+      branch => branch.name === branchName
+    )
```

Comment:
Nit: maybe extract this to a `getBranch` or `getBranchWithName` function and reuse in `onDeleteBranch`? 🤔